assets/js/app.js

```diff
@@ -34,9 +35,11 @@
                         today = parseInt(today);
 
                         if (comparisonDate >= today) {
+                            const day = new Date(holidays[i].date);
                             const nextHoliday = {
                                 name: holidays[i].name,
-                                date: new Date(holidays[i].date),
+                                date: day,
+                                weekDay: weekDays[day.getDay()],
```

Comment:
Daria também pra modernizar aqui tirando o array dos dias e trocando essa linha pela abaixo, então fica uma sugestão pra um possível update.
```suggestion
                                weekDay: new Intl.DateTimeFormat("pt-BR", { weekday: "long" }).format(day),
```